meta/saisanitycheck.c

```diff
@@ -3100,6 +3100,11 @@ void check_non_object_id_object_attrs()
 
             META_ASSERT_NOT_NULL(m);
 
+            if (m->isresourcetype)
+            {
+                continue;
```

Comment:
why this is added here? Do those resource types have different attributes ?

Reply:
Currently, non-object entries do not allow attributes other than CREATE_AND_SET. I introduced a READ_ONLY attribute to a route entry that tells the addr_family of the entry to be used to distinguish between different pools of resources in the function `sai_object_type_get_availability`
So this is needed to allow that kind of a READ_ONLY attribute of an entry.

And it already behaves that way - if absent, it is assumed that isresourcetype is false.

Reply:
yes, but your code allows all possible attributes, read_only, create_and_set etc, so please add extra limiting here theh when isresourcetype is true then attribute must be your expected type

Reply:
Done